Nef_3/include/CGAL/Nef_3/SNC_intersection.h

```diff
@@ -194,7 +107,7 @@ class SNC_intersection : public SNC_const_decorator<SNC_structure_> {
       return false;
     Vector_3 vs1(s1.to_vector()), vs2(s2.to_vector()),
```

Comment:
When I replace the next lines with
```
    Object o = intersection(s1, s2);
    return CGAL::assign(p,o);
```
and compute difference or intersection with the sphere grids it is faster.

Reply:
Yeah, I am also getting conflicting results. The first run of the benchmark, a modified version that uses google/benchmark, or just doing integration tests with sphere grids. I am confused.

Reply:
Can you give me (again) a link to your benchmark.

Reply:
This one? https://gist.github.com/GilesBathgate/043578b473e34be84435722c8735df41

Reply:
I think I prefer running Boolean operations on different types of data for figuring out which of the two is faster.

Reply:
Fair enough, I guess I was trying to confirm generally whether the author of SNC_intersection, had stumbled upon faster ways of doing intersections than the Kernel. I'd be surprised if that were the case, hence my original deletions.

Reply:
> When I replace the next lines with
> ...
> and compute difference or intersection with the sphere grids it is faster.


I did some more tests with sphere grids, which confirms what you say....
```
Baseline	98408f4115a
Time ms	14624
Time ms	15393
Time ms	15350
Time ms	15528
Time ms	15514
Time ms	15467
Time ms	16075
Time ms	15939
Time ms	15864
Time ms	15902
Mean secs	15.5656
Std-dev ms	418.70
	
Contender	e451af9f506
Time ms	14720
Time ms	14941
Time ms	15154
Time ms	15359
Time ms	15766
Time ms	15213
Time ms	15745
Time ms	15640
Time ms	15656
Time ms	15658
Mean secs	15.3852
Std-dev ms	366.78
	
(old-new)/old	1.16%
```